week2/04_log_and_register_custom_model.py

```
@@ -0,0 +1,136 @@
+from databricks.connect import DatabricksSession
+import mlflow
+import numpy as np
```

Ruff (0.7.0)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)